config/gregtech/worldgen/overworld/apatite_vein.json

```diff
@@ -1,27 +1,32 @@
 {
-  "weight": 60,
-  "density": 0.3,
-  "min_height": 60,
-  "surface_stone_material": "apatite",
-
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [13, 13]
-  },
-  "filler": {
-    "type": "simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 60,
-          "value": "ore:apatite"
-        },
-        {
-          "weight": 40,
-          "value": "ore:phosphor"
+    "weight": 60,
+    "density": 0.3,
+    "min_height": 60,
+    "surface_stone_material": "apatite",
+    "generator": {
+        "type": "ellipsoid",
+        "radius": [
+            13,
+            13
+        ]
+    },
+    "filler": {
+        "type": "simple",
+        "value": {
+            "type": "weight_random",
+            "values": [
+                {
+                    "weight": 60,
+                    "value": "ore:apatite"
+                },
+                {
+                    "weight": 40,
+                    "value": "ore:phosphor"
+                }
+            ]
         }
-      ]
-    }
-  }
+    },
+    "dimension_filter": [
```

Comment:
We usually have these at the top of the files, should probably standardize that for future viewing.

Reply:
I could do it, but it would be a pita to do as i have to do each file manually